frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.jsx

```diff
@@ -168,8 +168,11 @@ export default class TagEditorParam extends Component {
                 {() => (
                   <SchemaTableAndFieldDataSelector
                     databases={databases}
-                    selectedDatabaseId={database ? database.id : null}
-                    selectedTableId={table ? table.id : null}
+                    selectedDatabase={database || null}
+                    selectedDatabaseId={database?.id || null}
+                    selectedTable={table || null}
+                    selectedTableId={table?.id || null}
+                    selectedField={field || null}
```

Comment:
Pass the missing properties `selectedDatabase`, `selectedTable` and `selectedField` so on the first page/app load, the sidebar is rendered properly.